modules/go-add-infra-nodes.adoc

```diff
@@ -4,7 +4,21 @@
 
 :_content-type: PROCEDURE
 [id="add-infra-nodes_{context}"]
-= Adding infrastructure nodes
+= Moving workloads to infrastructure nodes
+
+The default workloads that can be moved to the infrastructure nodes are:
```

Comment:
This title and abstract seem incomplete. How about a tldr like this one:

```suggestion
= Moving GitOps infrastructure workloads to infrastructure nodes

To running GitOps control plane workloads on infrastructure nodes, you label existing nodes as infrastructure and move the workloads in the `openshift-gitops` namespace to them.

You can move the following default workloads to infrastructure nodes:
```

Reply:
@rolfedh changing it to Moving GitOps workloads to infrastructure nodes